### DISCUSSION TITLE
Release 2.0.48 with PHP8.1 fixes

### DISCUSSION THREAD

**Discussion by uaoleg** [yiisoft/yii2#19840] [General]
PHP8.1 is now minimum actively supported version. But it's still not possible to use it due to a number of known issues in Yii2 that were fixed in 2.0.48. 
So I propose to release 2.0.48 without fixes for PHP8.2. It's already is quite a big release.

**Comment by samdark** [comment]
Doable and makes sense. I'll move some issues to 2.0.49 milestone including PHP 8.2 support.

**Reply by samdark** [reply]
OK. Now the milestone is ready.
